client/src/components/itemCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Instrument } from "@/interfaces/instrument";
+import { Musician } from "@/interfaces/musician";
+import classNames from "classnames";
+import Image from "next/image";
+import { Typography } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+
+interface Props {
+  item: Musician | Instrument;
+  onClick: (item: Musician | Instrument) => void;
+}
+
+const useStyles = makeStyles(() => ({
+  card: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: "5%",
+    marginLeft: "2%",
+  },
+  selected: {
+    boxShadow: "0 0 0 5px #37A8FA",
+  },
+  image: {
+    width: 200,
+    height: 200,
+    overflow: "hidden",
+    borderRadius: "50%",
+    display: "flex",
+    justifyContent: "center",
+  },
+}));
+
+export const ItemCard = ({ item, onClick }: Props) => {
+  const handleClick = () => {
+    onClick(item);
+  };
+  const classes = useStyles();
+  return (
+    <div className={classNames(classes.card)}>
+      <div
+        className={classNames(classes.image, {
+          [classes.selected]: item.selected,
+        })}
+        onClick={handleClick}
+      >
+        <Image
+          priority
+          src={item.image}
+          alt={`select ${
+            "first_name" in item
```

Comment:
Do we need to use `in` here? Or can we just check for `item.first_name`? It's a very small change but avoids iterating through the item's properties

Reply:
When I tried checking for `item.first_name`, I got a type error, because `first_name` does not exist on type `Instrument`. I'm not sure how else to do this check, but would definitely be open to other ways! I agree that the loop through the properties is not efficient.

Reply:
Hmm weird. Well it’s definitely not a big deal so I say let’s merge it and if some brilliant idea occurs to one of us later we can always change it